ansible/roles/candlepin/vars/f35.yml

```diff
@@ -0,0 +1,19 @@
+# Variables used by various tasks in the role. These should only be updated with respect to changes
+# in the Candlepin repo itself, or the tasks surrounding its setup.
+---
+mariadb_packages:
+  - mariadb
+  - mariadb-java-client
+  - mariadb-server
+  - python3-PyMySQL
+mariadb_config_file: /etc/my.cnf.d/mariadb-server.cnf
+
+postgresql_packages:
+  - postgresql-server
+  - postgresql
+  - postgresql-jdbc
+  - python3-psycopg2
+postgresql_init_user: postgres
+postgresql_init_cmd: pg_ctl initdb -D /var/lib/pgsql/data
```

Comment:
The init command should be common between all the versions of postgresql we use from EL8 and beyond. In fact, without el7, the DB setup should be the same for everything.

If we're hitting a situation where we need to var out the entire thing, we're better off leaving the DB setup as individual tasks scoped for the OS.

Reply:
I accidentally copied vars files meant for EL9. Removed.

Reply:
ack